docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,50 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- Added new
+  [`experimentalSkipDomainInjection`](/guides/references/experiments#Experimental-Skip-Domain-Injection)
+  flag to disable Cypress from setting `document.domain` on injection. Addresses
```

Comment:
We should expand on what issue experiment is addressing? Also this is an e2e only config right?

Reply:
It is only an e2e config. I can add some details

Reply:
added more details and defined as e2e only in 2a4b1b2fa